test/c/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ foreach ( TEST ${TESTS} )
         PRIVATE ${CMAKE_SOURCE_DIR}/src
     )
     if ( T8CODE_FOUND )
-        target_include_directories( ${TARGET_NAME} PRIVATE ${T8CODE_INCLUDE_DIR} )
+        target_include_directories( ${TARGET_NAME} PRIVATE ${T8CODE_ROOT}/include/t8_fortran_interface/ )
```

Comment:
Why only include the fortran interface dir?

Reply:
Especially given that this is in the `c` directory

Reply:
Good catch, thanks!

With the new t8code cmake module, `target_link_libraries( ${TARGET_NAME} PRIVATE T8CODE::T8 )` magically seems to also add the corresponding include directories. The Fortran module however sits in a subdirectory, so I had to add this directory manually.

Anyways  it should not be needed here for the C tests.

Reply:
Verified, still works.